tests/visualization_tests/test_utils.py

```diff
@@ -173,3 +174,10 @@ def test_filter_inf_trials_message(caplog: LogCaptureFixture, with_message: bool
         assert n_filtered_as_inf == 1
     else:
         assert msg not in caplog.text
+
+
+def test_invalid_target_filter_nonfinite() -> None:
```

Comment:
This is easier to understand.
```suggestion
def test_filter_nonfinite_with_invalid_target() -> None:
```